README.md

```diff
@@ -107,11 +107,11 @@ be linked to the original voter.
     <img height="50px" src="docs/assets/audit-white.png#gh-dark-mode-only">
 </div>
 
-**Transparency/Verifiability/Auditability** - The whole election process is
+**Transparency/Verifiability/Auditability** - The whole form process is
```

Comment:
a "form process" doesn't make much sense, compared to the previous "election process". We should come with a better naming. In this case we might say the "voting process". 
(same in other places)